.coderabbit.yaml

```
@@ -0,0 +1,240 @@
+  # Code review depth
+  knowledge_base: true
+  enable_free_tier: true
+
+# Language and framework specific settings
+language:
+  typescript:
+    # TypeScript specific analysis
+    enable_strict_checks: true
+    check_type_safety: true
+    analyze_react_patterns: true
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)